--- gnomebaker.desktop.in.orig	2006-09-21 19:13:23.000000000 +0200
+++ gnomebaker.desktop.in	2006-09-21 19:19:44.000000000 +0200
@@ -2,12 +2,14 @@
 Name=GnomeBaker
 GenericName=CD/DVD Writer
 GenericName[da]=CD/DVD-brændeprogram
+GenericName[pl]=Program do tworzenia CD/DVD
 Comment=Write CD and DVD
 Comment[da]=Brænd CD og DVD
+Comment[pl]=Program do tworzenia CD/DVD
 Exec=gnomebaker
-Icon=gnomebaker-48
+Icon=gnomebaker.png
 Terminal=false
 Type=Application
-Categories=Application;AudioVideo;DiscBurning;
+Categories=GTK;GNOME;AudioVideo;DiscBurining
 StartupNotify=true
 Encoding=UTF-8
