diff -aurN gnomebaker-0.4.1.orig/gnomebaker.desktop.in gnomebaker-0.4.1/gnomebaker.desktop.in
--- gnomebaker-0.4.1.orig/gnomebaker.desktop.in	2005-08-14 23:34:07.000000000 +0200
+++ gnomebaker-0.4.1/gnomebaker.desktop.in	2005-08-16 16:48:48.000000000 +0200
@@ -1,11 +1,13 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=GnomeBaker
-Comment=GnomeBaker CD/DVD Writer
+Comment=Program for creating CDs/DVDs
+Comment[pl]=Program do tworzenia CD/DVD
 GenericName=CD/DVD Writer
 Exec=gnomebaker
-Icon=@PACKAGE_PIXMAPS_DIR@/gnomebaker-48.png
+Icon=gnomebaker.png
 Terminal=false
 MultipleArgs=false
 Type=Application
-Categories=Application;AudioVideo;DiscBurning;
+Categories=GTK;GNOME;AudioVideo;DiscBurning;
 StartupNotify=true
